lib/pavilion/resolver.py

```diff
@@ -166,6 +167,49 @@ def check_variable_consistency(self, raw_tests: List[Dict]):
                                 .format(test_name, test_suite, var_key, first_value_keys, i + 1,
                                         value_keys))
 
+    def filter_resolved_tests(self, resolved_tests, filters, outfile: IO[str] = None) -> List[ProtoTest]:
```

Comment:
I would rename this `select_permutations`, which makes it more clear what's actually happening.